Cargo.toml

```diff
@@ -12,6 +12,7 @@ description = """
 Easy support for interacting between JS and Rust.
 """
 edition = "2018"
+rust-version = "1.56"
```

Comment:
In any case, we can do this after this PR, let's not rebase this again.

Reply:
Setting `rust-version = "1.56"` is safe (not introducing any breakage), because wasm-bindgen-macro-support (an integral crate) depends on syn 2.0 which depends on edition 2021 which is provided since Rust 1.56. Therefore, we don't have to announce any MSRV constraint.

Setting the `rust-version` too low is safe and prevents lints that require more recent Rust versions (as you said). wasm-bindgen doesn't have a guaranteed MSRV, and changing this is orthogonal to this MR. If a too low rust version was confusing, we could also add a comment that wasm-bindgen isn't guaranteed to work on such old Rust versions.

Reply:
Yeah, you are right.